reference_guide/api_changes/api_changes_list_2020.md

```diff
@@ -68,10 +68,10 @@ Please see [Incompatible API Changes](/reference_guide/api_changes_list.md) on h
 
 ### Changes in IntelliJ Platform 2020.3
 
-`com.intellij.psi.PsiTypeElement`: PSI structure of multi-dimensional arrays in Java source files changed.
+The PSI structure of multi-dimensional arrays in Java source files changed (see `com.intellij.psi.PsiTypeElement`).
 Now the children are flattened: brackets for all the dimensions are direct children of the `PsiTypeElement` that represent the multi-dimensional array. This change doesn't break source or binary compatibility but may produce behavioral changes in the code that traverses the tree of Java source files.
```

Comment:
sorry missed, description must start with `:` (same below)

Reply:
Probably it's a good idea to automate these checks to avoid spending time on human review rounds?